packages/mdc-tab/_variables.scss

```diff
@@ -28,5 +28,5 @@ $mdc-tab-text-label-opacity: .6 !default;
 $mdc-tab-icon-opacity: .54 !default;
 $mdc-tab-text-label-color-default: rgba(mdc-theme-prop-value(on-surface), $mdc-tab-text-label-opacity) !default;
```

Comment:
Should these `mdc-theme-prop-value` calls be updated, too?

Reply:
Since we're mixing the color with alpha here we've to set color value to this variable.